src/js/veteran-representative/config/form.js

```diff
@@ -0,0 +1,282 @@
+import fullSchema from '../2122-schema.json';
+import _ from 'lodash/fp';
+import IntroductionPage from '../containers/IntroductionPage';
+import ConfirmationPage from '../containers/ConfirmationPage';
+
+import fullNameUI from '../../common/schemaform/definitions/fullName';
+import ssnUI from '../../common/schemaform/definitions/ssn';
+import * as addressUI from '../definitions/address';
```

Comment:
This should probably use the address definition from `src/js/common/schemaform/definitions`